ciao-controller/internal/datastore/datastore_test.go

```diff
@@ -797,45 +797,45 @@ func TestGetBatchFrameSummary(t *testing.T) {
 
 	err := ds.HandleTraceReport(trace)
 	if err != nil {
-		t.Error(err)
+		t.Fatal(err)
 	}
 
 	_, err = ds.db.getBatchFrameSummary()
 	if err != nil {
-		t.Errorf(err.Error())
+		t.Fatalf(err.Error())
```

Comment:
same in multiple from here forward...why sometimes "err" and sometimes "err.Error()"?